modular_skyrat/modules/customization/game/objects/items/tanks/n2_tanks.dm

```diff
@@ -26,7 +27,7 @@
 	inhand_icon_state = "nitrogen"
 	slot_flags = ITEM_SLOT_BELT
 	force = 5
-	volume = 24
+	volume = 12 // changed from previous 24, its the same as double oxy. I think it was suposed to be double oxy equivalent suggesting icon state name.
```

Comment:
```suggestion
	volume = 12
```

Reply:
Remember to change the default kPA setting to match the reduced need, make it the same as emergency oxygen.

Reply:
> Remember to change the default kPA setting to match the reduced need, make it the same as emergency oxygen.

Default kpa is already set on all tanks sharing the typepath to 4 kpa